djoser/urls/authtoken.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import re_path
 
-from djoser import views
+from djoser.views.token.create import TokenCreateView
+from djoser.views.token.destroy import TokenDestroyView
 
 urlpatterns = [
-    re_path(r"^token/login/?$", views.TokenCreateView.as_view(), name="login"),
-    re_path(r"^token/logout/?$", views.TokenDestroyView.as_view(), name="logout"),
+    re_path(r"^token/login/?$", TokenCreateView.as_view(), name="login"),
+    re_path(r"^token/logout/?$", TokenDestroyView.as_view(), name="logout"),
```

Comment:
We may want to switch to `url()` instead of `re_path` by the way of doing this views split.

Reply:
https://github.com/sunscrapers/djoser/pull/820/commits/44ea180c7d52235a1820d40517a8f90855ea3a80